src/VisualStudio/Core/Test/ProjectSystemShim/VisualBasicHelpers/MockCompilerHost.vb

```diff
@@ -20,6 +20,12 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.ProjectSystemShim.Vi
             End Get
         End Property
 
+        Public Shared ReadOnly Property NoSdkCompilerHost As MockCompilerHost
```

Comment:
As a general rule I just assume the spec will be violated at some point in the future :smile_cat: